### ISSUE TITLE
Font Awsome Icons not appearing  - Moodle 3.9.1

### ISSUE THREAD

**Issue by ghost** [moodleuulm/moodle-local_boostnavigation#68]
Having followed the documentation I've created a link using the exemplar ending in...   /course/view.php?id=2||||||fa-graduation-cap

This does not display the expected icon but a generic box.
![image](https://user-images.githubusercontent.com/5968088/89886608-cac1e880-dbc4-11ea-8675-86337be9c718.png)

This is a superb plug in by the way; hope it becomes core.

**Comment by abias** [comment]
Hey @demsley ,

on my 3.9 test instance, custom icons work flawlessly.

Could you please double check the syntax of this custom node configuration? I think you got something wrong.

Cheers,
Alex

**Comment by ralie29** [comment]
Hi! I'm having a similar issue on 2 moodle 3.8.2+ instances: (Build: 20200402 and Build: 20200408)
I use the example given in the comments here: https://moodle.org/plugins/local_boostnavigation

```
Site administration quicklinks|/admin/search.php|en
-List of users|/admin/user.php|||||fa-users
-Manage courses|/course/management.php|||||fa-graduation-cap
-Capability overview|/admin/tool/capability/index.php|||||fa-check-square-o
-Notifications|/admin/index.php|||||fa-wrench
```

The result:
![image](https://user-images.githubusercontent.com/73182729/96602941-3bd7f780-12fc-11eb-8e9c-520eac83d520.png)

It looks the same in all browsers.
Do you have any ideas why this isn't working?

**Comment by abias** [comment]
I am sorry, but this example from the plugin comments seems to be wrong. There is one pipe missing. It should be:
```
Site administration quicklinks|/admin/search.php|en
-List of users|/admin/user.php||||||fa-users
-Manage courses|/course/management.php||||||fa-graduation-cap
-Capability overview|/admin/tool/capability/index.php||||||fa-check-square-o
-Notifications|/admin/index.php||||||fa-wrench
```

**Comment by ralie29** [comment]
Hey Alexander! Thanks for the quick response!
We are one step further with the extra pipe but the icons still can't be properly displayed:
![image](https://user-images.githubusercontent.com/73182729/96700323-fe28ac80-1397-11eb-8207-d017fbcac976.png)
Any additional help would be greatly appreciated :)

**Comment by abias** [comment]
Hey @ralie29 ,

now that the 3.9 release of this plugin is finally out, I have tested the reported issue again on 3.9 - and I was not able to reproduce it. However, I reopened the issue just to be sure.

Could you please test again, ideally on a fresh installation of Moodle 3.9, with the v3.9-1 release of the plugin? 
If you can reproduce this issue, please provide steps to reproduce for me.

Thanks,
Alex

**Comment by abias** [comment]
Hey @ralie29 ,

having just published the 3.10 version of the plugin, I am still not able to reproduce this issue on 3.9 and not on 3.10.

With the configuration
```
Site administration quicklinks|/admin/search.php|en
-List of users|/admin/user.php||||||fa-users
-Manage courses|/course/management.php||||||fa-graduation-cap
-Capability overview|/admin/tool/capability/index.php||||||fa-check-square-o
-Notifications|/admin/index.php||||||fa-wrench
```
all custom nodes appear fine:
![grafik](https://user-images.githubusercontent.com/1092118/101949261-9baf9800-3bf3-11eb-84a4-bc4611db5a46.png)

So I think it's time to close this issue again until you come back to me with steps to reproduce.

Cheers,
Alex
